games/service_igdb.py

```diff
@@ -0,0 +1,130 @@
+from ast import List
+from requests import post
+import json
+
+
+class MyAuth:
+    """
+    Main goal is get token, in order to dos, we need credentilas(client_id, secret
+    """
+
+    def __init__(self, secret: str, client_id: str, url: str):
+        self.secret = secret
+        self.client_id = client_id
+        self.url = url
+
+    def get_token(self, grant_type: dict):
+        """
+        1. Create post request
+        2. Provide cretentials and grant_type(applicable for twitch
+        """
+        auth_body = {
+            "client_id": self.client_id,
+            "client_secret": self.secret,
+        }
+        auth_body = auth_body | grant_type
+        auth_response = post(self.url, auth_body)
+        return auth_response.json()
+
+
+# igdb
+URL_TWITCH = "https://id.twitch.tv/oauth2/token"
+client_secret_twitch = "hz9akahuzsa418xmflcfqrpdoqvokq"
+client_id_twitch = "txzxbs4vpf2y9scp1045n6oyzqzwxm"
```

Comment:
let's take this offline, it's a bad practice to store these value in a code. During a call today, I'll show you how to handle this